snc.sh

```diff
@@ -223,26 +223,30 @@ create_pvs "${CRC_PV_DIR}" 30
 # https://github.com/openshift/cluster-version-operator/blob/master/docs/dev/clusterversion.md#setting-objects-unmanaged
 ${OC} patch clusterversion version --type json -p "$(cat cvo_override.yaml)"
 
-# Get the pod name associated with cluster-monitoring-operator deployment
 cmo_pod=$(${OC} get pod -l app=cluster-monitoring-operator -o jsonpath="{.items[0].metadata.name}" -n openshift-monitoring)
```

Comment:
have you considered using `-oname` instead?

Reply:
@cfergeau yes we can use that, I will put another PR for that.